WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewModel.swift

```diff
@@ -256,7 +256,8 @@ extension ReaderTabViewModel {
             return
         }
 
-        WPAnalytics.track(.readerFilterSheetCleared)
+        let type = ((activeStreamFilter?.topic as? ReaderSiteTopic) != nil) ? "site" : "topic"
```

Comment:
Minor nitpick, could we use `is` here instead?
```suggestion
        let type = activeStreamFilter?.topic is ReaderSiteTopic ? "site" : "topic"
```

Reply:
Good idea. Thanks!